### PR TITLE
O3-1489 Adding links to view lists in clinic metrics

### PR DESCRIPTION
**Pull request by CynthiaKamau**
…ent module

## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
- Added scheduled appointment link to help view scheduled appointments list

## Screenshots

https://user-images.githubusercontent.com/19533785/187650197-fbada625-555b-4714-8b89-c8f6fd4c1dda.mp4

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-appointments-app/src/config-schema.ts [hunk 1]
```diff
@@ -26,6 +26,11 @@ export const configSchema = {
     _description: 'Configurable days of the week',
     _default: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'],
   },
+  scheduledAppointmentListUrl: {
+    _type: Type.String,
+    _default: 'appointments-list/scheduled',
+    _description: 'URL to load the scheduled appointment list',
+  },
```
[the remark this answers is not in the document]
Comment (CynthiaKamau) [reply packages/esm-appointments-app/src/config-schema.ts:33]: I put it here just incase there is someone a different queue module

File: packages/esm-appointments-app/src/appointments-metrics/metrics-header.component.tsx [hunk 2]
```diff
@@ -13,8 +13,8 @@ const MetricsHeader: React.FC = () => {
       <Button
         renderIcon={(props) => <ArrowRight size={16} {...props} />}
         kind="ghost"
-        iconDescription={t('moreMetrics', 'See more metrics')}>
-        {t('moreMetrics', 'See more metrics')}
+        iconDescription={t('seeMissedAppointments', 'See Missed Appointments')}>
+        {t('seeMissedAppointments', 'See Missed Appointments')}
```
[the remark this answers is not in the document]
Comment (CynthiaKamau) [reply packages/esm-appointments-app/src/appointments-metrics/metrics-header.component.tsx]: Done

File: packages/esm-appointments-app/src/config-schema.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (CynthiaKamau) [reply packages/esm-appointments-app/src/config-schema.ts]: removed it

**Comment by CynthiaKamau** [comment]
Please review @makombe

File: packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx [hunk 3]
```diff
@@ -27,17 +27,20 @@ const AppointmentsMetrics: React.FC = () => {
           label={t('patients', 'Patients')}
           value={totalAppointments}
           headerLabel={t('scheduledAppointments', 'Scheduled appointments')}
+          view="patients"
         />
         <MetricsCard
           label={t(highestServiceLoad?.serviceName)}
           value={highestServiceLoad?.count ?? '--'}
           // FIX: Strange translation string
           headerLabel={t('highestServiceVolume', 'High volume Service.  today')}
```
Comment (denniskigen) [inline packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx:36]: This doesn't affect this PR, but what should this translation string read as? `High volume Service. today` seems a bit off. Or is this just a placeholder?

**Review by denniskigen — APPROVED** [review]
A small note about an unrelated thing, otherwise looks good.

File: packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx (hunk 3 above)
Comment (CynthiaKamau) [reply packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx:36]: its a placeholder

**Pull request merged**